LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Buf Technologies, Inc.
+   Copyright 2023-205 Buf Technologies, Inc.
```

Comment:
```suggestion
   Copyright 2023-2025 Buf Technologies, Inc.
```

Reply:
TY! My keyboard is giving up the ghost :/